### DISCUSSION TITLE
Welcome to wsdd Discussions!

### DISCUSSION THREAD

**Discussion by christgau** [christgau/wsdd#74] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rokawl** [comment]


**Reply by christgau** [reply]
Could you please create a new discussion [here](https://github.com/christgau/wsdd/discussions) and write your message inside the editor that is presented here? A PDF file is not really a good place to start a discussion. 

Thanks.

**Comment by rokawl** [comment]
Hi Steffen,
Thank you, not use to the proper syntax first time.
Bob,
 
--------- Original Message --------- Subject: Re: [christgau/wsdd] Welcome to wsdd Discussions! (#74)

**Reply by rokawl** [reply]
# Command-line options for wsdd
OPTIONS="-4 -i eth0 -i eth1 -w WORKGROUP"

#\fB\-n \fIHOSTNAME\fR, \fB\-\-hostname \fIHOSTNAME\fR
 Override the host name wsdd uses during discovery. By default the machine's host name is used (look at hostname(1)
 
I am working with ClearOS ver 7.7.2  I installed wsdd, It put the top two lines in Wsdd ./etc/sysconfig and detected host clearOS machine name but not the Windows Networking (Samba) which normally is  a different host name, it looks like I should Put it in something like this
 OPTIONS="-n DINW, --HOSTNAME dinw  ?
 DINW being the Samba host/NETBIOS name and dinsrv being the ClearOS server network host name. Was able to get it working by giving them both the same name not sure what will happen If used this way. But not a proper fix.

I was able to get it working right by adding -n HOSTNAME which is the netbios name in the smb,config file, replacing the command-line options for wsdd.
 OPTIONS="-4 -i eth0 -i eth1 -w WORKGROUP -n DINW"

 Thank You
Bob

**Reply by christgau** [reply]
> [something] detected host clearOS machine name but not the Windows Networking (Samba) which normally is a different host name, 

wsdd does not interact with samba directly. If the name that is configured in Samba should be used, then whoever launches wsdd must pass that hostname to wsdd...

> it looks like I should Put it in something like this
OPTIONS="-n DINW, --HOSTNAME dinw ?

... by exactly that option. Note that specifying either `-n` or `--HOSTNAME` is sufficient.

> I was able to get it working right by adding -n HOSTNAME which is the netbios name in the smb,config file,

Fine. We obviously came to the same solution 😉
